apps/web/src/app/dashboard/[name]/overview/page.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { SearchIcon } from "lucide-react";
+import { StretchHorizontal } from "lucide-react";
+import { usePathname } from "next/navigation";
+import { useState } from "react";
+
+import VideoCard from "~/components/dashboard/videoCard";
+import VideoSmallCard from "~/components/dashboard/videoSmallCard";
+import { Button } from "~/components/ui/button";
+import { Input } from "~/components/ui/input";
+import { Toggle } from "~/components/ui/toggle";
+import mockdata from "~/lib/mock/organizationOverview";
+import { api } from "~/trpc/react";
+
+export default function Dashboard() {
+  // TODO: replace mocks with db queries
+  const { data: organizations } =
```

Comment:
This value is never used. For now change this to:
```suggestion
const { data: _organizations } =
```